db/app.js

```diff
@@ -22,8 +25,17 @@ app.get("/api/articles", getArticles);
 
 app.get("/api/:article_id/comments", getCommentsByArticleId);
 
+app.post("/api/articles/:article_id/comments", postCommentByArticleId);
+
+if (process.env.NODE_ENV !== "test") {
```

Comment:
it's better practice to extract this out to its own file as you've seen in the lectures